Remotion/Web/Core/Res/Themes/NovaViso/HTML/WebButton.css

```diff
@@ -2,105 +2,54 @@
 
 button.webButton
 {
-  background-color: Transparent;
-  cursor: hand;
+  background-color: transparent;
+  border: var(--remotion-themed-border-width) solid var(--color-primary);
+  border-radius: var(--remotion-themed-border-radius);
+  padding: 0.4em 1.2em;
```

Comment:
one more thing: please check that setting the width on the button control or setting the width va css on button-element works correctly (i.e. horizontally centers the text)